internal/querycoordv2/checkers/segment_checker.go

```diff
@@ -192,13 +192,14 @@ func (c *SegmentChecker) getGrowingSegmentDiff(collectionID int64,
 			continue
 		}
 
+		nextTargetExist := c.targetMgr.IsNextTargetExist(collectionID)
```

Comment:
1. why don't we just skip this round if nextTarget not Exist?
2. targetVersion := c.targetMgr.GetCollectionTargetVersion(collectionID, meta.CurrentTarget)
		if view.TargetVersion != targetVersion {
			// before shard delegator update it's readable version, skip release segment
			log.RatedInfo(20, "before shard delegator update it's readable version, skip release segment",
				zap.String("channelName", channelName),
				zap.Int64("nodeID", node),
				zap.Int64("leaderVersion", view.TargetVersion),
				zap.Int64("currentVersion", targetVersion),
			)
			continue
		}
can we handle by this logic above but not introduce another nextTargetExist check? 
the code itself is very redundant

Reply:
1. we can't use same condition to skip this round, cause load segment just require next target exist, but release segment require both current target and next target exists.
2. target version can't block releasing segment in next target, cause it only works for current target